src/main/scala/mesosphere/marathon/core/CoreModuleImpl.scala

```diff
@@ -53,7 +54,8 @@ class CoreModuleImpl @Inject() (
   scheduler: Provider[DeploymentService],
   instanceUpdateSteps: Seq[InstanceChangeHandler],
   taskStatusUpdateProcessor: TaskStatusUpdateProcessor,
-  homeRegionProvider: Provider[HomeRegionProvider]
+  homeRegionProvider: Provider[HomeRegionProvider],
+  @Named(ModuleNames.ELECTION_EXECUTOR) electionExecutor: ExecutorService,
```

Comment:
Why not pass it?

Reply:
We can define executor in application.conf and then get it from actor system